golang/_redirects

```diff
@@ -49,7 +48,6 @@
 /kn-plugin-source-kamelet/* go-get=:go-get /golang/kn-plugin-source-kamelet.html 200
 /knobots/* go-get=:go-get /golang/knobots.html 200
 /monitoring/* go-get=:go-get /golang/monitoring.html 200
-/net-certmanager/* go-get=:go-get /golang/net-certmanager.html 200
```

Comment:
we shouldn't drop some of these old redirects - since it will break older clients

I wonder if we should just not do this for archived repos

Reply:
Ok I can fix it in another PR.